corpus_explorer/utils.py

```diff
@@ -198,3 +203,69 @@ def generate_topic_scatter_plot(topic_coordinates, topic_proportions):
             marker_size=topic_sizes,
         ),
     )
+
+
+def generate_visualization(topic_coordinates, topic_proportions):
```

Comment:
I assume at some point one of the inputs here will be the pandas df for the corpus? At the very least we will want to pass some sort of structure that contains the metadata for each document so that we can subset directly through the dash app. This is good for the base framework to build on.

Reply:
NVM you addressed this in the PR comment.